iothub/device/src/InternalClient.cs

```diff
@@ -1273,6 +1290,130 @@ public Task<Message> ReceiveAsync(CancellationToken cancellationToken)
             }
         }
 
+        /// <summary>
```

Comment:
same comment as above here and everywhere

Reply:
which comment?